### PR TITLE
Add ageur details

### PR DESCRIPTION
**Pull request by tutacrypto**

### CODE REVIEW

File: guides/ageur-power-usecase.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Get paid in agEUR, receive funds in your bank account
+
+![Untitled](/.gitbook/assets/angle-borrowmodulewp-announcement-light.png)
+
+Thanks to its powerful partners and integrations, agEUR can now be used to pay and create invoices, and agEUR can be easily transferred to your home bank account. All of this from almost anywhere in the world, on Ethereum or Polygon!
```
Comment (sogipec) [inline guides/ageur-power-usecase.md:5]: agEUR can be used to pay and create invoices.

Goal of the docs is to be permanent, not like an article, and so better not to use "now"

File: guides/ageur-power-usecase.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+This can be very useful on two sides: 1) for companies or DAOs wanting to pay their contributors in crypto, and 2) for contributors happy to be paid in crypto but who still need fiat in a bank account for their everyday lives. For example, some DAOs might not have a bank account, or just prefer to pay their contributors in crypto. However, these contributors usually need a proper proof of payment (invoice) and a way to transfer those funds to their bank account easily, without taking care of the exchange rate. This is all possible with agEUR today: anyone can start invoicing and getting paid in agEUR, and transfer them directly to EUR in their bank account!
```
Comment (sogipec) [inline guides/ageur-power-usecase.md:7]: Can skip a line before the for example

Comment (sogipec) [reply guides/ageur-power-usecase.md:7]: This is all possible with agEUR today: we should reformulate it in a way that makes it less "article"

File: guides/ageur-power-usecase.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+Here, we try to show how companies, DAOs, and their contributors can leverage agEUR and Angle’s partners  Request, Mt Pelerin, and Superfluid to do so quickly and in a compliant way. 
+
+## Invoicing in agEUR with Request
+
+### Contributors side
+
+- [Request](https://app.request.finance/)
+
+Request lets users create and send custom invoices to their clients. Users can denominate their invoice in crypto or fiat, and ask to be paid in crypto or fiat. For example, a german or french resident working for an international DAO could label their invoice in EUR, but ask to be paid in agEUR.
```
Comment (sogipec) [inline guides/ageur-power-usecase.md:17]: a German or French

File: guides/ageur-power-usecase.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+![request create invoice](/.gitbook/assets/request-create-invoice.png)
+
+The invoice will look like that 👇
+
+![Request invoice](/.gitbook/assets/request-invoice.png)
+
+### Crypto Company / DAO side: pay contributor with agEUR from any token
+
+On the crypto company side, once they received the invoice they can just send the required amount of agEUR to the specified address. If they don’t own agEUR yet, they can go through Angle’s app at [app.angle.money](http://app.angle.money), and add the invoicer address as recipient. Then, they can select **any token they hold**, and use the expert mode to make sure the minimum amount received is that of the invoice.
```
Comment (sogipec) [inline guides/ageur-power-usecase.md:29]: On the company side

Comment (sogipec) [reply guides/ageur-power-usecase.md:29]: Once they receive (not received)

File: guides/ageur-power-usecase.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+- **contributors** to **invoice crypto companies and get paid in agEUR** in a fully compliant way, and
+- **contributors** to **transfer agEUR** **to EUR in their bank account** to cover the cost of their everyday lives!
+
+That should be enough to help many companies and DAOs to pay their contributors, and those contributors to receive funds in their local bank account! But there is more ... streaming money incoming 💸👇
+
+![agEUR power use case](/.gitbook/assets/agEUR%20power%20use%20case.png)
+
+## 💸 Go further by streaming those payments with Superfluid 💸
+
+Superfluid allows user to **stream tokens on Polygon and xDAI**. Thanks to Angle’s agEUR integration, users can receive their payments not as a one-time transaction, but through **a payment stream** using Superfluid. This will allow them to get paid continuously, and not just once at the end of each month.
```
Comment (sogipec) [inline guides/ageur-power-usecase.md:59]: This allows them to get paid continuously

Comment (sogipec) [reply guides/ageur-power-usecase.md:59]: No need to use future here

File: core-module/lending.md [hunk 6]
```diff
@@ -72,5 +72,10 @@ This makes it both more efficient that other folding strategies, and a clear imp
 As of April 2022, a single folding strategy for USDC on Aave has been implemented.
 
 {% hint style="info" %}
-The state of the strategies used for different collateral types can be tracked on [Angle Analytics](https://analytics.angle.money).
+The state of the strategies used for different collateral types can be tracked on the [Angle Analytics](https://analytics.angle.money).
 {% endhint %}
+
+### stETH strategy
+
+The strategy used for ETH is the StETHAcc strategy from Yearn. It compares rates for stETH between Lido and the Curve stETH/ETH pool, and earn the stETH native yield. More complex ETH strategies will be developed in the future. The contract can be found [here](https://github.com/AngleProtocol/angle-core/blob/main/contracts/strategies/StrategyStETHAcc.sol).
```
[the remark this answers is not in the document]
Comment (tutacrypto) [reply core-module/lending.md:80]: What do you mean by a ref to the Yearn strategy?

File: guides/getAgEUR.md [hunk 7]
```diff
@@ -6,67 +6,95 @@ description: How to mint and burn agTokens using Angle app
...
 
-- mint new agEUR directly from the [Angle App](https://app.angle.money/) against your tokens
-- swap ETH for agEUR on [Uniswap](https://app.uniswap.org) or any DEX aggregator on Ethereum mainnet or on any supported chain.
-- buy agEUR with credit card from one of our on-ramp partners like [Wert.io](https://widget.wert.io/01FPZ4G1VMZ67MGC83NADPB0F8/redirect/?commodity=agEUR&commodities=agEUR,agEUR%3Aethereum)
-
-## Buying agEUR
+## 🔎 TL;DR
 
-**With an on-ramp solution** If you're not familiar with Ethereum, you can create a wallet with [Ledger](https://www.ledger.com) or [Metamask](https://metamask.io) and go through our on-ramp partner at [wert.io](https://widget.wert.io/01FPZ4G1VMZ67MGC83NADPB0F8/redirect/?commodity=agEUR&commodities=agEUR,agEUR%3Aethereum) to buy agEUR on Ethereum or Polygon.
+There are multiple ways to buy agEUR:
```
[the remark this answers is not in the document]
Comment (tutacrypto) [reply guides/getAgEUR.md:14]: The idea was that it's good to get as much people through our frontend as possible, but I agree it's not ideal if we don't do it as well as others

File: guides/getAgEUR.md [hunk 8]
```diff
@@ -6,67 +6,97 @@ description: How to mint and burn agTokens using Angle app
...
-### Burning agEUR
+### Selling agEUR
 
-If you want to get rid of your agEUR, you have a few options:
+If you want to exchange your agEUR back to another token, you have a few options:
 
-- swap them against another token on [Uniswap](https://app.uniswap.org)
-- burn agEUR against one of the available tokens from the [Angle App](https://app.angle.money/#/user/burn).
+- Swap them against another token directly on [Angle](https://app.angle.money/#/user/burn) or a decentralized exchange like [Uniswap](https://app.uniswap.org). 
+- Send money back directly to your bank account through our off-ramp parner Mt. Pelerin.
```
Comment (sogipec) [inline guides/getAgEUR.md:51]: off-ramp partner Mt Pelerin

**Review by sogipec — APPROVED** [review]

**Pull request merged**